source/includes/debt-settlement.md.erb

```diff
@@ -587,31 +586,42 @@ Content-Type: application/json
+ - Finally the system will publish a <a href="/#webhooks-investment_repayment_batch_complete">INVESTMENT_REPAYMENT_BATCH_COMPLETE</a> when the batch in considered complete. This is a batch process and therefore might be sent later than the last repayment to clear.
+ - All these webhooks will reference `repaymentBatchReference` so you are required to store this value for reconciling repayments as they clear.
 
 <aside class="notice">
 This endpoint is the only call required to repay investments for any platform onboarded from 1st November 2021.
 </aside>
 
 ### Request
-| Name                              | Type   | Description                                                                    | Required |
-| --------------------------------- | ------ | ------------------------------------------------------------------------------ | -------- |
-| reference                         | string | The payment reference used for the repayment of funds. Will also be present on `INVESTOR_FUNDS_RECEIVED` webhook | required |
-| sourceWalletId                    | string | The wallet this distribution should be made from                               | required |
-| investorRepayments                | array  | The repayments per investor.                                                   | required |
-| investorRepayments[].investmentId | string | The ID of the of the investment                                                | required |
-| investorRepayments[].type         | string | The type of the repayment Possible values are: <br>`CAPITAL`<br>`INTEREST`<br> | required |
-| investorRepayments[].amount       | ref    | The amount being repaid                                                        | required |
-| investorRepayments[].tax          | ref    | The amount of tax being withheld from this repayment                           | optional |
+| Name                              | Type   | Description                                                                                            | Required |
+| --------------------------------- | ------ | ------------------------------------------------------------------------------------------------------ | -------- |
+| reference                         | string | The payment reference used for the repayment of funds. This will be present on the wallet statement.   | required |
+| sourceWalletId                    | string | The wallet this distribution should be made from                                                       | required |
+| investorRepayments                | array  | The repayments per investor.                                                                           | required |
+| investorRepayments[].investmentId | string | The ID of the of the investment as specified in the original investment creation call                  | required |
```

Comment:
Hyperlink to the endpoint?

Reply:
No hyperlink?

Reply:
I think this might be a GitHub bug

Reply:
No this was me doing it in one place and not in the other 😵‍💫